modin/core/execution/ray/common/utils.py

```diff
@@ -64,7 +120,15 @@ def initialize_ray(
         What password to use when connecting to Redis.
         If not specified, ``modin.config.RayRedisPassword`` is used.
     """
-    extra_init_kw = {"runtime_env": {"env_vars": {"__MODIN_AUTOIMPORT_PANDAS__": "1"}}}
+    extra_init_kw = {
+        "runtime_env": {
+            "env_vars": {
+                "__MODIN_AUTOIMPORT_PANDAS__": "1",
+                "MODIN_LOG_MODE": LogMode.get(),
+                "MODIN_LOG_FILE_SIZE": str(LogFileSize.get()),
```

Comment:
I would rather you only pass environment variables if they have non-default values, as this value is shown to the user in a warning